src/registrar/models/domain_information.py

```diff
@@ -57,6 +57,16 @@ class Meta:
         help_text="Person who submitted the domain request",
     )
 
+    # portfolio
+    portfolio = models.OneToOneField(
```

Comment:
**blocking** This is a "many domains to one portfolio" aka a many to 1 relationshipt between domain info and portfolio. One to one would make it so no portfolio can have more than one domain info, which doesn't work for the org model. This needs to be a ForeignKey field in django.